core/src/main/resources/error/error-classes.json

```
@@ -469,6 +469,11 @@
       "Grouping sets size cannot be greater than <maxSize>"
     ]
   },
+  "GROUP_BY_AGGREGATE" : {
+    "message" : [
+      "aggregate functions are not allowed in GROUP BY, but found <sqlExpr>"
```

The error message should begin from an upper case letter and ends with a dot. See other error messages in `error-classes.json`
```suggestion
      "Aggregate functions are not allowed in GROUP BY, but found <sqlExpr>."
```